setup.sh

```diff
@@ -37,6 +37,7 @@ clear(){
 install(){
   echo 'Installing project dependencies...'
   npm install
+  npm install protractor@2.1.0 -g
```

Comment:
I guess [.travis.yml](https://github.com/cfpb/cfgov-refresh/blob/flapjack/.travis.yml#L24-L25) is the best place for protractor?